pom.xml

```diff
@@ -48,6 +48,11 @@
     </dependency>
 
     <dependency>
+      <groupId>se.sawano.java</groupId>
```

Comment:
Same dependency as we have for npm for doing the same basic comparison, which should (to my knowledge) also be suitable for this scenario (or at least no worse than it is for npm).